src/class-fields/yield-var.case

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: yield var
+info: |
+  1.1 New Productions
+
+  [...]
+
+  FieldDefinitionList [Yield, Await]:
+    FieldDefinition [?Yield, ?Await]
+    FieldDefinitionList  [?Yield, ?Await], FieldDefinition [?Yield, ?Await]
+template: default
+includes: [propertyHelper.js]
+---*/
+
+//- body
+  yield
```

Comment:
The grammar has `[?Yield, ?Await]`, but doesn't appear to define what to do with those productions... I wonder if @littledan intends to define something similar to https://tc39.github.io/ecma262/#sec-identifiers-static-semantics-early-errors